filetests/isa/x86/simd-comparison-run.clif

```diff
@@ -231,3 +231,17 @@ ebb0:
     return v8
 }
 ; run
+
+function %fcmp_gt_nans_f32x4() -> b1 {
+ebb0:
+    v0 = vconst.f32x4 [NaN 0x42.0 -NaN NaN]
+    v1 = vconst.f32x4 [NaN NaN 0x42.0 Inf]
```

Comment:
@julian-seward1, it's not exhaustive but this is more what you're looking for?

Reply:
Yes, that's better.  Thanks.